jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PushBlobStep.java

```diff
@@ -86,18 +86,7 @@ public BlobDescriptor call() throws IOException, RegistryException, ExecutionExc
         return blobDescriptor;
       }
 
-      // If base and target images are in the same registry, then use mount/from to try mounting the
-      // BLOB from the base image repository to the target image repository and possibly avoid
-      // having to push the BLOB. See
-      // https://docs.docker.com/registry/spec/api/#cross-repository-blob-mount for details.
-      boolean sameRegistry =
-          buildConfiguration
-              .getBaseImageConfiguration()
-              .getImageRegistry()
-              .equals(buildConfiguration.getTargetImageConfiguration().getImageRegistry());
-      String mountFrom =
-          sameRegistry ? buildConfiguration.getBaseImageConfiguration().getImageRepository() : null;
-      registryClient.pushBlob(blobDescriptor.getDigest(), blob, mountFrom);
+      registryClient.pushBlob(blobDescriptor.getDigest(), blob, null);
```

Comment:
Let's add a TODO for considering adding mount? Just so we have a placeholder for if we want to possibly add the feature back in in the future.